governator-test-junit/src/main/java/com/netflix/governator/guice/test/CopyBindingTargetVisitor.java

```diff
@@ -0,0 +1,81 @@
+package com.netflix.governator.guice.test;
+
+import java.lang.reflect.Constructor;
+
+import com.google.inject.Scopes;
+import com.google.inject.binder.LinkedBindingBuilder;
+import com.google.inject.spi.BindingTargetVisitor;
+import com.google.inject.spi.ConstructorBinding;
+import com.google.inject.spi.ConvertedConstantBinding;
+import com.google.inject.spi.ExposedBinding;
+import com.google.inject.spi.InstanceBinding;
+import com.google.inject.spi.LinkedKeyBinding;
+import com.google.inject.spi.ProviderBinding;
+import com.google.inject.spi.ProviderInstanceBinding;
+import com.google.inject.spi.ProviderKeyBinding;
+import com.google.inject.spi.UntargettedBinding;
+
+public class CopyBindingTargetVisitor<T> implements BindingTargetVisitor<T, Void> {
```

Comment:
Do you really want this to be part of the supported public API?  If not, consider making it package private.